features/proxy.feature

```diff
@@ -39,51 +39,27 @@ Scenario: Authenticated HTTP proxy without creds
     Then I wait for 5 seconds
     And I should receive no requests
 
-@skip_agp4_0_or_higher
```

Comment:
These tags can probably be removed throughout the tests and where they're defined in the Ruby file.